packages/ember-toucan-core/package.json

```diff
@@ -33,7 +33,7 @@
   },
   "peerDependencies": {
     "@crowdstrike/ember-toucan-styles": "^2.0.1",
-    "@ember/test-helpers": "^2.8.1",
+    "@ember/test-helpers": "^2.8.1 || ^3.0.0",
```

Comment:
I found it strange as to why this was listed as a `peerDependency`, but then stumbled upon https://github.com/CrowdStrike/ember-toucan-core/blob/main/packages/ember-toucan-core/src/test-support/components/button.ts (which I added at the time 🙈).  Due to us exposing some test helpers, we need this (unless we'd rather move it to a `dependency`, I suppose). 

We may also want to decide if we want to increase test-support in the future by providing more fractal-page-objects for our other components, now that we have quite a few.

Reply:
Yeah, I think this seems correct then. You _could_ probably declare this peerDep as optional, on the other hand basically every app already has a dependency on `@ember/test-helpers`, so that doesn't matter much I guess...